subworkflows/nf-core/bam_docounts_contamination_angsd/tests/main.nf.test.snap

```diff
@@ -0,0 +1,76 @@
+{
+    "homo_sapiens - bam bai - hapmap -- stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "1": [
+                    "versions.yml:md5,3bf0fb169d6ae7f35569cafd1d904f6e",
+                    "versions.yml:md5,564bdd3eda2bfc9e585010991e3451f7"
+                ],
+                "txt": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "versions": [
+                    "versions.yml:md5,3bf0fb169d6ae7f35569cafd1d904f6e",
+                    "versions.yml:md5,564bdd3eda2bfc9e585010991e3451f7"
+                ]
+            },
+            [
+                {
+                    "BAM_DOCOUNTS_CONTAMINATION_ANGSD:ANGSD_CONTAMINATION": {
+                        "angsd": "0.940-dirty"
+                    }
+                },
+                {
+                    "BAM_DOCOUNTS_CONTAMINATION_ANGSD:ANGSD_DOCOUNTS": {
+                        "angsd": 0.939
```

Comment:
Hmm, this is different to above.

Reply:
I tried to update all the tools to the same version but if I do so everything crash 😢

Reply:
Fine,  not your problem